src/test/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/testfixtures/domain/timeline/TimeLineBuilder.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.testfixtures.domain.timeline
```

Comment:
This file really needs to live in the `testFixtures` source root, rather than in the `test` source root in a package called `testfixtures` !!!

There's some weird conflict between the standard gradle plugin `java-test-fixtures`, and something in the DPS Spring Boot gradle plugin. I fell into a bit of a time sink rabbit hole last night trying to get to the bottom of it !!!, so for now we'll have it here in the `test` source root, and we can look to work out what is going on some other time